core/src/main/kotlin/utils/JobMetrics.kt

```diff
@@ -113,4 +174,94 @@ class JobMetrics(private val application: Application) : MeterBinder {
 
     private fun countScannerJobs(status: JobStatus) =
         transaction { ScannerJobDao.count(ScannerJobsTable.status eq status).toDouble() }
+
+    private fun getAnalyzerJobDurations(timestamp: Instant): List<Duration> =
+        transaction {
+            AnalyzerJobsTable.selectAll().where {
+                    AnalyzerJobsTable.status eq JobStatus.FINISHED or
+                            (AnalyzerJobsTable.status eq JobStatus.FINISHED_WITH_ISSUES)
+                }.andWhere {
+                    AnalyzerJobsTable.startedAt.isNotNull()
+                }.andWhere {
+                    AnalyzerJobsTable.finishedAt.isNotNull()
+                }.andWhere {
+                    AnalyzerJobsTable.finishedAt greaterEq timestamp
+            }.mapNotNull {
+                val startedAt = it[AnalyzerJobsTable.startedAt] ?: return@mapNotNull null
+                val finishedAt = it[AnalyzerJobsTable.finishedAt] ?: return@mapNotNull null
+                finishedAt - startedAt
+            }
+        }
+
+    private fun getAdvisorJobDurations(timestamp: Instant): List<Duration> =
+        transaction {
+            AdvisorJobsTable.selectAll().where {
+                    AdvisorJobsTable.status eq JobStatus.FINISHED or
+                            (AdvisorJobsTable.status eq JobStatus.FINISHED_WITH_ISSUES)
+                }.andWhere {
+                    AdvisorJobsTable.startedAt.isNotNull()
+                }.andWhere {
+                    AdvisorJobsTable.finishedAt.isNotNull()
+                }.andWhere {
+                    AdvisorJobsTable.finishedAt greaterEq timestamp
+            }.mapNotNull {
+                val startedAt = it[AnalyzerJobsTable.startedAt] ?: return@mapNotNull null
+                val finishedAt = it[AnalyzerJobsTable.finishedAt] ?: return@mapNotNull null
```

Comment:
This should be `AdvisorJobsTable.startedAt` and `AdvisorJobsTable.finishedAt`, similar for the functions below.